config/src/config/mod.rs

```diff
@@ -454,5 +459,9 @@ mod test {
         NodeConfig::default_for_public_full_node();
         NodeConfig::default_for_validator();
         NodeConfig::default_for_validator_full_node();
+
+        let contents = std::include_str!("test_data/safety_rules.yaml");
```

Comment:
nit: if we give safety rules it's own config name constant, this string can be derived from that name (see my comment above)

Reply:
well, those config constants are only defined in config-builder, no? not that it doesn't make sense to probably move everything to libra-global-constants.